gallery/src/components/NxGlobalSidebar/NxGlobalSidebarExample.tsx

```diff
@@ -37,14 +48,24 @@ export default function NxGlobalSidebarExample() {
                                          text="NxPagination"
                                          href="#/pages/NxPagination"/>
         </NxGlobalSidebarNavigation>
-        <section className="gallery-custom-sidebar-content nx-global-sidebar__expanded-content">
+        <section className="gallery-custom-sidebar-content
+                            nx-global-sidebar__other-content
+                            nx-global-sidebar__expanded-content">
           <NxH3>
             Custom Content
           </NxH3>
           <NxP>
             neural decay saturation point assault camera neon concrete engine
           </NxP>
         </section>
+        <NxGlobalSidebarFooter showSupport={true}
+                               supportText="Support for RSC"
+                               supportLink="https://github.com/sonatype/sonatype-react-shared-components"
+                               releaseText="React Shared Components"
+                               releaseNumber={packageJson.version}
```

Comment:
I'd rather just hardcode something here for the example

Reply:
> I'd rather just hardcode something here for the example

https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/b974a4ebcab15cdc3edfe587c17fd6c0e546c2fe